README.md

```diff
@@ -59,7 +59,37 @@ spec:
 
 This will create a new Sandbox named `my-sandbox` running the image you specify. You can then access the Sandbox using its stable hostname, `my-sandbox`.
 
-For more complex examples, including how to use the extensions, please see the [examples/](examples/) and [extensions/examples/](extensions/examples/) directories.
+For a more programatic approach you can use the [SDK](https://github.com/kubernetes-sigs/agent-sandbox/blob/main/clients/python/agentic-sandbox-client/README.md) like the following:
+```python
+from agentic_sandbox import Sandbox
+
+# The SDK abstracts all YAML into a simple context manager
+try:
+    with Sandbox(
+        template_name="python-runtime-template",
+        namespace="ai-agents"
+    ) as sandbox:
+
+        print("--- Sandbox is Ready! ---")
+
+        # 1. Run a command inside the secure sandbox
+        result = sandbox.run("echo 'Hello from inside the sandbox!'")
+        print(f"Stdout: {result.stdout.strip()}")
+
+        # 2. Write and read files
+        sandbox.write("test.txt", "This is a test file.")
+        content = sandbox.read("test.txt").decode('utf-8')
+        print(f"Read content: {content}")
+
+except Exception as e:
+    print(f"An error occurred: {e}")
```

Comment:
Catching the generic `Exception` is a bit broad. If the SDK has its own exception types (e.g., for connection errors, sandbox creation failures), it would be better to catch those specifically. If not, a comment explaining that this is simplified for the example would be helpful.